src/backend/cgelem.c

```diff
@@ -5443,4 +5448,89 @@ void postoptelem(elem *e)
     }
 }
 
+/***********************************
+ * Rewrite rvalues of complex numbers to pairs of floating point numbers.
+ */
+static elem *elToPair(elem *e)
+{
+    switch (e->Eoper)
+    {
+        case OPvar:
+        {
+#if 0
```

Comment:
So what's the deal with this stuff? If you ever want that code back it's in git history. How about just remove it?